Remotion/Core/Reflection/NullMethodInformation.cs

```diff
@@ -28,22 +28,32 @@ namespace Remotion.Reflection
   /// </summary>
   public sealed class NullMethodInformation : IMethodInformation
   {
-    public string Name
+    public string? Name

```

Comment:
The null-implementations need a revamp. The fact that the not-nullablity has been ignored is a bug.

Reply:
What should i do with those, the interfaces declare something as not nullable and the implementation returns null. NFOing everything seems dangerous.

Reply:
Right now, that's what we will do. The fact that we need to change this to something better is the bugfix-todo&jira-ticket